cmd/plugin/commands/certs/certs.go

```diff
@@ -46,7 +46,9 @@ func CreateCommand(flags *genericclioptions.ConfigFlags) *cobra.Command {
 	}
 
 	cmd.Flags().String("host", "", "Get the cert for this hostname")
-	cobra.MarkFlagRequired(cmd.Flags(), "host")
+	if err := cobra.MarkFlagRequired(cmd.Flags(), "host"); err != nil {
```

Comment:
I think this could be ignored or exited (os.Exit(1)). An error on Cobra is a bug in our program and should be raised for us as an issue :)

Reply:
I agree, the error should be exposed if it's a bug.